api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -141,6 +141,8 @@ public void setUp() {
 
     providedWorkbenchConfig = WorkbenchConfig.createEmptyConfig();
     providedWorkbenchConfig.accessRenewal.expiryDays = (long) 365;
+    providedWorkbenchConfig.access.enableComplianceTraining = true;
```

Comment:
new-style is more sensitive to feature flags, so tests were failing before this change